### DISCUSSION TITLE
Example of TDD counter app

### DISCUSSION THREAD

**Discussion by meysammahfouzi** [FilledStacks/stacked#1007] [Q&A] [answered]
I am interested in moving towards TDD, but I really don't know how to start. Could you please create a tutorial for that? It would be great if in that tutorial, you create a new app using the stacked CLI, write the tests that fail, and then write the code for the counter application. Thanks!

**Comment by ferrarafer** [comment, marked as answer]
Hi @meysammahfouzi , how are you? That should probably be a suggestion for @FilledStacks 😄 . At the moment, we are very busy with some high priority tasks but I think [this](https://www.filledstacks.com/post/better-unit-tests-in-flutter-using-abstraction/) post could be a good starting point for what you want to achieve.

Cheers!
